### ISSUE TITLE
Default implementations for Elements?

### ISSUE THREAD

**Issue by Keyslam** [Keyslam/Inky#6]
In my opinion, it is not possible to create a set of widgets that work for everyone's usecase, without being way to bloated and complicated to use.
However, I also recognize that some users just want some basic UI elements they can set up, and be done with it.

I am debating if Inky should provide some default implementation for elements, such as buttons, labels, sliders, checkboxes, radioboxes, text fields, et cetera. And if so, how would these widgets be styled by the user?

Alternatively, these widgets could be hosted in one or more other repositories, there could be a Material-like implementation, a ImGUI-like implementation, et cetera.

**Comment by badduck32** [comment]
No idea if this is an issue that you want feedback on, but I feel like the second implementation would be the way to go. This way inky itself stays non-bloated, but if people want some examples or quick and easy implementations of Inky to import to their project, they would have those repositories to rely on.

I don't know how useful my advice is though since I haven't used Inky that much, but I am using it on my current project so I can give an update with some more concrete ideas in the future, and maybe even help set up those other repositories mentioned

**Comment by pablomayobre** [comment]
I agree with badduck, and would like to add that you should try to track down projects using Inky to show its capabilities and (if open source) link to their usage

**Comment by pakeke-constructor** [comment]
I'd be happy to start on some of this
